external-crates/move/crates/move-compiler/src/typing/match_compilation.rs

```diff
@@ -1,2916 +0,0 @@
-// Copyright (c) The Move Contributors
```

Comment:
Whoops, embarrassing that this got left in

Reply:
It happens :) I only found it because of regex